### ISSUE TITLE
missing enum groups

### ISSUE THREAD

**Issue by 3b** [KhronosGroup/OpenGL-Registry#109]
following are used in in function parameter definitions but don't have a `group` definition:

```
BinormalPointerTypeEXT
CombinerComponentUsageNV
CombinerPortionNV
CombinerStageNV
CullParameterEXT
DataTypeEXT
ElementPointerTypeATI
EvalMapsModeNV
FragmentLightNameSGIX
FragmentLightParameterSGIX
GetVariantValueEXT
IndexFunctionEXT
IndexMaterialParameterEXT
MapTypeNV
MatrixIndexPointerTypeARB
ParameterRangeEXT
PathColorFormat
PixelTransformPNameEXT
PixelTransformTargetEXT
ProgramFormat
ProgramStringProperty
ProgramTarget
ReplacementCodeTypeSUN
ScalarType
SecondaryColorPointerTypeIBM
TangentPointerTypeEXT
VariantCapEXT
VertexShaderTextureUnitParameter
VertexShaderWriteMaskEXT
VertexWeightPointerTypeEXT
WeightPointerTypeARB
```

(updated 2019-06-21 in response to recent PRs - @oddhack)

**Comment by oddhack** [comment]
Please propose a PR (or maybe several PRs for different vendor's groups - this looks like a ton of work and it's more difficult to evaluate giant scattershot PRs).

**Comment by 3b** [comment]
Yeah, will probably be at most a few at a time if at all, not even a whole vendor.  Not sure how many would still be useful for new code, and old ones are fairly low priority.

**Comment by Perksey** [comment]
opentk/opentk#903 is blocked by this.

**Comment by oddhack** [comment]
@Perksey we'd welcome any PRs you're able to contribute along these lines. The enum group stuff is purely for the benefit of downstream consumers - we don't use it at all for Khronos purposes - so there's not a lot of motivation to spend time on this.

**Comment by Perksey** [comment]
Ah, alright. I'll try to direct new contributors to this issue, we're stuck on the 26 Dec 2017 version of the spec for the time being :|

**Comment by Perksey** [comment]
Can someone at Khronos update this issue, removing the enum groups added in #273 and #274?

**Comment by oddhack** [comment]
@Perksey I believe it's up to date now.

**Comment by Perksey** [comment]
Awesome - whoof that's still a lot of missing groupings.

**Comment by frederikja163** [comment]
Look, I would be happy to update the enums and put a lot of hours into making them work again. The only problem I have is that I frankly don't know how I would fix them. I do not know which files to look through/which documentation. So if the guys at Khronos know how to fix it but simply don't have the time. Please do point me in the right direction and I will definitely do my best to get them all fixed

**Comment by frederikja163** [comment]
Update: I spent the last 5 hours looking at the spec and extensions folder, and now i should have them all fixed, i will make a PR right away

**Comment by Perksey** [comment]
This issue can be closed now in response to #292.

**Comment by 3b** [comment]
Thanks, that mostly works for me, with a few changes in pull #332.
